bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-uk-2.html

```diff
@@ -0,0 +1,83 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('firefox_whatsnew_105_eu') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-whatsnew-105-eu') %}
+    {{ js_bundle('firefox_whatsnew_105_uk_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% set utm_campaign = campaign + '-uk-discover' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md mzp-l-split-pop-top mzp-t-content-xl wnp-content-main',
+    body_class='mzp-l-split-v-start',
+    media_class='mzp-l-split-h-center',
+    image_url='img/firefox/whatsnew/whatsnew105-eu/discover-uk-500.png',
+    image_srcset={
+      'img/firefox/whatsnew/whatsnew105-eu/discover-uk-500.png': '500w',
+      'img/firefox/whatsnew/whatsnew105-eu/discover-uk-700.png': '700w',
+      'img/firefox/whatsnew/whatsnew105-eu/discover-uk-900.png': '900w',
+    },
+    image_sizes={
+      '(min-width: 1312px)': '458px',
+      '(min-width: 768px)': 'calc(50vw - 192px)',
+      'default': 'calc(100vw - 48px)'
+    },
+    image_height='638',
+    image_width='458',
+    image_alt='Articles from a wide range of publications',
+    media_after=True
+  ) %}
+    <div class="mzp-c-emphasis-box wnp-main-content">
+      <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-pocket"></div>
+      <h2 class="wnp-main-title">Discover what inspires you</h2>
```

Comment:
**style nit (non-blocking)**: one hanging word on large screen headline

**suggestion**: use of `br` tag as in `-fr` template, or maybe a little extra right padding on large screen headline to ensure "inspires" bumps to second line

<img width="592" alt="Screen Shot 2022-09-12 at 2 05 54 PM" src="https://user-images.githubusercontent.com/19650432/189661698-daa4d0d2-a179-4592-9ba1-a85fc0ea3d58.png">